tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -421,6 +406,9 @@ def is_verbose(argv: argparse.Namespace):
 
 
 def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
+    start_time = datetime.datetime.now()
+    tracemalloc.start()
```

Comment:
Can we measure memory only when verbose=True

Reply:
Yes, done.